projects/ad7606x_fmc/zed/system_project.tcl

```diff
@@ -10,21 +10,43 @@ source $ad_hdl_dir/projects/scripts/adi_board.tcl
 # Parameter description
 # DEV_CONFIG - The device which will be used
 #  - Options : AD7606B(0)/C-16(1)/C-18(2)
+# INTF - Operation interface
+#  - Options : Parallel/Serial
```

Comment:
Can we update the description to something like: Parallel(0)/Serial(1) and maybe add a description similar to the one described [here](https://github.com/analogdevicesinc/hdl/blob/main/projects/cn0506/README.md#building-generating-bit-files) for all the parameters.

Reply:
Done.